src/elements/Svg.tsx

```diff
@@ -43,6 +43,11 @@ export interface SvgProps extends GProps, ViewProps, HitSlop {
   preserveAspectRatio?: string;
   color?: ColorValue;
   title?: string;
+  toDataURL?: (
+    svgString: string,
+    callback: (data: string) => void,
+    options?: { width?: number; height?: number }
+  ) => void;
```

Comment:
It's a function on component ref, not a property

Reply:
Thank you, I removed that.